Enterprise-CMCS/macpro-mdct-mcr
# Mdct 2864 yoy specific

### Description

This PR is to update the YoY feature to only copy over the specific fields requested by stakeholders (can be found in the document linked in the ticket)

### Related ticket(s)


MDCT-2864

---
### How to test

- create and submit a MCPAR report
- create a new MCPAR report, copying that previously submitted report
- Go through the copied report and verify against the document linked in the Jira ticket


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
